examples/qml_features/rust/src/containers.rs

```diff
@@ -33,7 +33,7 @@ pub mod qobject {
     }
 
     unsafe extern "RustQt" {
-        #[cxx_qt::qobject(qml_uri = "com.kdab.cxx_qt.demo", qml_version = "1.0")]
+        #[cxx_qt::qobject(qml_element)]
```

Comment:
This is nicer than repeating that copypasta over and over :smiley:

Reply:
:tada: